package.json

```diff
@@ -64,7 +65,7 @@
         "typescript": "^4.8.4"
     },
     "peerDependencies": {
-        "eslint": ">=6.6.0"
+        "eslint": "^6.6.0 || ^7.0.0"
```

Comment:
I don't think this change is necessary.
And, I think this change to be a breaking change as it means it can no longer be used with v8.

Reply:
Technically it's a breaking change, but we're also only testing for `^6.6.0 || ^7.0.0`, so we also don't _really_ support other versions

I'll revert this & include it in the ESLint v8 support PR instead 👍